src/quo/components/list_items/network_list_item/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns quo.components.list-items.network-list-item.view
```

Comment:
best to keep the ns mapping 1-1 with Figma.
i.e ` quo.components.list-items.network-list.view`

Reply:
I originally had it named as `network-list`, but decided to change it to `item`. A list would suggest a list of items, whereas this is an item in a list. 

Happy to update to follow the figma convention

Reply:
same for style and component spec

Reply:
nice, yeah ideally we ask the designers to change it. Either way it's easier to maintain the design system if we keep it 1-1 as it's much quick to look up etc 👌

Reply:
sounds good, will update